src/cores/account/BotAccount.sol

```diff
@@ -0,0 +1,68 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.13;
+
+import {Account} from "src/cores/account/Account.sol";
+import {BaseAccount} from "src/cores/account/BaseAccount.sol";
+import {IEntryPoint} from "src/lib/ERC4337/interface/IEntryPoint.sol";
+import {ValidatorsStorage} from "src/validator/ValidatorsStorage.sol";
+import {Initializable} from "src/lib/initializable/Initializable.sol";
+import {Ownable} from "src/access/ownable/Ownable.sol";
+import {OwnableInternal} from "src/access/ownable/OwnableInternal.sol";
+import {Access} from "src/access/Access.sol";
+import {Operations} from "src/lib/Operations.sol";
+
+/// @title Station Network Bot Account Contract
+/// @author 👦🏻👦🏻.eth
+
+/// @dev This contract provides a single hub for managing and verifying signatures
+/// created by addresses with the `Operations::CALL_PERMIT` permission.
+/// ERC1271 and ERC4337 compliance in combination with the 0xRails permissions system
+/// provides convenient and modular private key management on an infrastructural level
+contract BotAccount is Account, Initializable {
```

Comment:
Per an earlier comment, I would expect the inheritance of `Ownable` here because other non-abstract Account implementations use a different ownership scheme.

Reply:
Gotcha, resolved in cc6de26